tools/ci-cdk/canary-lambda/ci-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+# Run by CI to check the canary-lambda
+set -e
+cd "$(dirname $0)"
+
+SDK_PATH="$(git rev-parse --show-toplevel)"/aws/sdk/build/aws-sdk/sdk
+if [[ "${GITHUB_ACTIONS}" == "true" ]]; then
+   SDK_PATH="$(git rev-parse --show-toplevel)"/aws-sdk/sdk
+fi
+
+./write-cargo-toml.py --path "${SDK_PATH}"
+cargo check
```

Comment:
I don't think we need to run both? also, can this use `additional-ci` instead?

Reply:
Yeah, I think you're right about not needing to run both. I will address this and attempt to consolidate it into `additional-ci` as a follow-up pull request to unblock the release.

Reply:
Addressed in https://github.com/awslabs/smithy-rs/pull/1084